Samples/SimpleStructures/Program.cs

```diff
@@ -58,26 +58,23 @@ public CustomDataType(int value)
         /// </summary>
         static void Main()
         {
-            using (var context = Context.CreateDefault())
+            using var context = Context.CreateDefault();
+
+            // For each available device...
+            foreach (var device in context)
             {
-                // For each available device...
-                foreach (var device in context)
-                {
-                    // Create accelerator for the given device
-                    using (var accelerator = device.CreateAccelerator(context))
-                    {
-                        Console.WriteLine($"Performing operations on {accelerator}");
-                        var kernel = accelerator.LoadAutoGroupedStreamKernel<Index1D, ArrayView<CustomDataType>>(MyKernel);
-                        using (var buffer = accelerator.Allocate1D<CustomDataType>(1024))
-                        {
-                            // Launch buffer.Length many threads and pass a view to buffer
-                            kernel((int)buffer.Length, buffer.View);
+                // Create accelerator for the given device
+                using var accelerator = device.CreateAccelerator(context);
+                Console.WriteLine($"Performing operations on {accelerator}");
+
+                var kernel = accelerator.LoadAutoGroupedStreamKernel<Index1D, ArrayView<CustomDataType>>(MyKernel);
+                using var buffer = accelerator.Allocate1D<CustomDataType>(1024);
+
+                // Launch buffer.Length many threads and pass a view to buffer
+                kernel((int)buffer.Length, buffer.View);
 
-                            // Wait for the kernel to finish...
-                            accelerator.Synchronize();
-                        }
-                    }
-                }
+                // Wait for the kernel to finish...
+                accelerator.Synchronize();
```

Comment:
Remark: this synchronization is different and important to avoid crashes. Maybe we should explain some more stuff here?